src/runtime/vm/vm.cc

```diff
@@ -1057,7 +1057,7 @@ runtime::Module CreateVirtualMachine(const Executable* exec) {
   return runtime::Module(vm);
 }
 
-TVM_REGISTER_GLOBAL("relay._vm._VirtualMachine")
+TVM_REGISTER_GLOBAL("runtime._vm._VirtualMachine")
```

Comment:
consider just make it ```runtime.VirtualMachine```

Reply:
then it will be available under runtime._ffi_api

Reply:
Sure. Will update in a bit.